Directory.Build.props

```diff
@@ -23,22 +23,12 @@
   </PropertyGroup>
 
   <!--
-    Arcade only allows the revision to contain up to two characters, and GitHub Actions does not roll-over
-    build numbers every day like Azure DevOps does. To balance these two requirements, set the official
-    build ID to be the same format as the built-in default from Arcade, except with the revision number
-    being the number of the quarter hour of the current time of day (24 * 4 = 96, which is less than 100).
-    So a build between 00:00 and 00:14 would have a revision of 1, and a build between 23:45 and 23:59:59
-    would have a revision of 97.
+    These are set per-project so versioning is applied correctly, but are not set globally otherwise
```

Comment:
Oh? Interesting. Well, I guess that answers my question 😄

Reply:
Yeah, that's what I tried first but it go 💥 😅

https://github.com/martincostello/AspNet.Security.OAuth.Providers/runs/836762272?check_suite_focus=true#step:5:301